src/stores/account.ts

```diff
@@ -245,4 +250,23 @@ export default class AccountStore implements IAccountStore {
     this._privateKeyToRecovery = null;
     return account;
   };
+
+  fixKeyLength = async (v: ProtectedPrivateKey, passphrase: string) => {
+    const key = V3toRaw(await fs.promises.readFile(v.path, "utf8"), passphrase);
+    if (key.length < 32) {
+      fs.promises.writeFile(
+        v.path,
+        (
+          await new Wallet(
+            new SigningKey(
+              Buffer.concat([Buffer.alloc(32 - key.length), key], 32)
+            )
+          ).encrypt(passphrase)
+        )
+          .replace(/"id":"[0-9A-z-]*"/, `"id":"${v.keyId}"`)
+          .replace("C", "c")
```

Comment:
This was added due to encrypt method of `ethers` is generating json with uppercase "Crypto", which should be lowercase.

Reply:
To represent intention, how about replacing `"Crypto"` to `"crypto"`?